apify-api/openapi/paths/actor-runs/actor-runs@{runId}@charge.yaml

```diff
@@ -0,0 +1,46 @@
+post:
+  tags:
+    - Actor runs/Charge events in run
+  summary: Charge events in run
+  description: |
+    Charge for events in the run of your [pay per event Actor](https://docs.apify.com/platform/actors/running/actors-in-store#pay-per-event).
+    The event you are charging for must be one of the configured events in your Actor. If the Actor is not set up as pay per event, or if the event is not configured,
+    the endpoint will return an error. The endpoint must be called from the Actor run itself, with the same API token that the run was started with.
+
+    Note that pay per events Actors are still in alpha. Please, reach out to us with any questions or feedback.
+  operationId: PostChargeRun
+  parameters:
+    - name: runId
+      in: path
+      required: true
+      schema:
+        type: string
+      example: 3KH8gEpp4d8uQSe8T
+      description: Run ID.
+    - name: idempotency-key
```

Comment:
I proposed in my previous comment using `x-` prefix for the header name should it remain but I removed the comment. I just learned that [`rfc6648`](https://www.rfc-editor.org/rfc/rfc6648) deprecates the convention of `x-` prefixing the non-standard headers and all introduced header names should be treated as they might become standard. So my opinion is that the right way of introducing a custom header would be perhaps `ppe-` or `apify-` prefix which sets a clear bounded context for the slightly wide domain term `idempotency-key`.

I'm happy to elaborate in case it's needed!

Reply:
Clarified over Slack by @mhamas pointing out the original design document and backend implementation thought process.